layouts/partials/card.html

```diff
@@ -46,14 +47,26 @@
           {{ if $customTitle }}
             <div>{{ $customTitle }}</div>
           {{ else }}
-            <div>{{ .LinkTitle }}</div>
-          {{ end }}
-          {{ if ne $class "small" }}
-          <p> {{ if $customDescription }} {{ $customDescription }} {{ else }} {{ .Description }} {{ end }} </p>
+            {{ errorf "Couldn't get images %s" .Params.images }}
```

Comment:
this doesn't make sense here. Something went wrong with the merge and because it's a merge commit, it has changed the history in ways I'm not super keen to untangle.

Reply:
I recommend you do a hard reset to before the merge conflict and then pull rebase and have a closer look at what changed and how the changes should now fit together.

Reply:
Yeah, yesterday I was trying to merge and resolve conflicts but there were mega conflicts with the recent change to this file so was gonna work more on fixing today

Reply:
I did manage to come up with something that merges the logic when trying to rebase, If you do want to copy that from [here](https://github.com/npentrel/viam-docs/blob/0b5e49a0f088d0d7ffaea2be2484f52a96574837/layouts/partials/card.html) you can. I believe somehow some stuff gets duplicated even during the rebase. A bit odd.